src/i18n/locales/en.json

```diff
@@ -94,5 +94,22 @@
 		"themes": "Themes",
 		"discord": "Discord",
 		"github": "Github"
+	},
+	"download": {
+		"title": "Download Files",
+		"donation_description": "Please consider donating to support our work on Files",
+		"donation_button": "Donate",
+		"microsoft_store": {
+			"title": "Microsoft Store",
```

Comment:
"Microsoft Store" and "Files" should be kept in English.

Reply:
Which "Files" text do you mean?

Reply:
Anytime it refers to the app name

Reply:
I think maintaining that using template strings is a lot more effort than just watching out nobody translates files in these strings. For example "Download Files" is "Files herunterladen" in german, so we would need to template string if we don't want to includes files in those strings. Or maybe I missunderstood what you mean.

Reply:
Template strings would work

Reply:
Should we do that for all texts in that file or only for the download ones?

Reply:
We should really do it for all of them

Reply:
There doesn't seem to be a very elegant way but it's done now.